pkg/model/sdk_helper.go

```diff
@@ -168,8 +168,8 @@ func (a *SDKAPI) GetOperationMap(cfg *ackgenconfig.Config) *OperationMap {
 	return &opMap
 }
 
-// Given an API operation and member of API operation, return shape reference associated with the member
-func (a *SDKAPI) GetMemberShapeRef(operation string, memberName string) (*awssdkmodel.ShapeRef, bool) {
+// Given an API operation and member of API operation, return shape reference associated with the input member
```

Comment:
nit: by convention comments should begin with the name of the item they describes

Reply:
Updated code comments.